gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,16 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-47068   
**Check Name:** rollup: DOM Clobbering Gadget found in rollup bundled scripts that leads to XSS   
**Severity:** HIGH   
**Fixed Version:** 3.29.5, 4.22.4, 2.79.2   
**Reachable Path(s) Found:** No  
**Description:** Rollup is a module bundler for JavaScript. Versions prior to 3.29.5 and 4.22.4 are susceptible to a DOM Clobbering vulnerability when bundling scripts with properties from `import.meta` (e.g., `import.meta.url`) in `cjs`/`umd`/`iife` format. The DOM Clobbering gadget can lead to cross-site scripting (XSS) in web pages where scriptless attacker-controlled HTML elements (e.g., an `img` tag with an unsanitized `name` attribute) are present. Versions 3.29.5 and 4.22.4  contain a patch for the vulnerability.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-47068